azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/model/implementation/CreatableImpl.java

```diff
@@ -33,32 +33,6 @@ protected CreatableImpl(String name, InnerModelT innerObject) {
     }
 
     /**
-     * create this resource and creatable resources it depends on.
-     * <p>
-     * dependency resources will be created only if this is the root group otherwise
-     * it creates the main resource.
-     *
-     * @throws Exception the exception
-     */
-    protected void creatablesCreate() throws Exception {
-        if (creatableTaskGroup.isRoot()) {
-            creatableTaskGroup.prepare();
-            creatableTaskGroup.execute();
-        } else {
-            createResource();
-        }
-    }
-
-    protected ServiceCall creatablesCreateAsync(ServiceCallback<Void> callback) {
-        if (creatableTaskGroup.isRoot()) {
-            creatableTaskGroup.prepare();
-            return creatableTaskGroup.executeAsync(callback);
-        } else {
-            return createResourceAsync(callback);
-        }
-    }
-
-    /**
      * add a creatable resource dependency for this resource.
```

Comment:
same capitalization issue as earlier. I won't comment on the others -- it'd be good to just do a scrub.
I wonder if our checktyle rules could enforce caps in the first sentence too, in addition to that period we already enforce..